src/hb-ot-cff1-table.hh

```diff
@@ -40,6 +40,7 @@ namespace CFF {
 #define HB_OT_TAG_cff1 HB_TAG('C','F','F',' ')
 
 #define CFF_UNDEF_SID   CFF_UNDEF_CODE
+#define HB_CFF_STD_STR_COUNT	391
```

Comment:
This also magic.  Should rewrite in terms of the actual array size or something else.

Reply:
I see in `hb-ot-post-table.hh`, the value of `NUM_FORMAT1_NAMES` is hard-coded to 258 when it could be calculated in `hb-string-array.h`:
```
#define HB_STRING_ARRAY_LENGTH_NAME	HB_PASTE(HB_STRING_ARRAY_NAME, _length)
static const unsigned HB_STRING_ARRAY_LENGTH_NAME = (ARRAY_LENGTH(HB_STRING_ARRAY_OFFS_NAME)-1);
```
Are you suggesting something like this?

Reply:
Sure, that would work.  Please submit that separately first.

Reply:
created PR 2055